### DISCUSSION TITLE
Login and forgot password with custom field

### DISCUSSION THREAD

**Discussion by bocanhcam** [laravel/nova-issues#6348] [Q&A]
Some of my applications will have an ID or username beside an email for users.

Is it possible to use forgot password or log in with username or ID instead of an email?

- Laravel Version: 10
- Nova Version: 4
- PHP Version: 8.2

**Comment by webard** [comment]
You can install Breeze or Jetstream or implement own logic for auth, it is not dependent from Nova.
